workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/src/Generation/Model/Details/UnityFieldDetails.cs

```diff
@@ -1,44 +1,68 @@
-using System.Collections.Generic;
-using Improbable.CodeGeneration.Model;
-using Improbable.CodeGeneration.Utils;
+using System;
+using System.Collections.ObjectModel;
+using Improbable.Gdk.CodeGeneration.Model.SchemaBundleV1;
```

Comment:
`SchemaBundleV1` - V1? is this generated?

Reply:
This matches the bundle specification - I guess the implication is that at some point we have have `SchemaBundleV2` and they could exist side by side